### ISSUE TITLE
[Bug]: Failing GithubActions

### ISSUE THREAD

**Issue by davidrsch** [Appsilon/box.lsp#28]
### Guidelines

- [X] I agree to follow this project's Contributing Guidelines.

### Project Version

0.1.1

### Platform and OS Version

 Ubuntu   22.04.4   LTS
### What happened?

Having this code in the .Rprofile

```
options(
  languageserver.parser_hooks = list(
    "box::use" = box.lsp::box_use_parser
  )
)
```

Yields the following error in github actions:

```
Error: Error in loadNamespace(x) : there is no package called ‘box.lsp’
``` 

I remove it and tests run as expected

### Steps to reproduce

Push a rhino app into github after running `box.lsp::use_box_lsp()` to configure your project's .Rprofile file. 

### Expected behavior

That jobs run without errors or with other error 🤣

**Comment by radbasa** [comment]
Hi @davidrsch , what version of `{rhino}` are you using?

**Comment by davidrsch** [comment]
I am using `{rhino 1.10.0}`

**Comment by radbasa** [comment]
Could you check if `box.lsp` is in the `dependencies.R` and `renv.lock` files?

**Comment by davidrsch** [comment]
Yes I have it in both files, I installed the package as suggested in description `rhino::pkg_install("box.lsp")` and the i executed `box.lsp::use_box_lsp()` which added:
```
options(
  languageserver.parser_hooks = list(
    "box::use" = box.lsp::box_use_parser
  )
)
```
to my .Rprofile which in turns made Github Actions return error:
```
Error: Error in loadNamespace(x) : there is no package called ‘box.lsp’
```
I eliminated:
```
options(
  languageserver.parser_hooks = list(
    "box::use" = box.lsp::box_use_parser
  )
)
```
from my .Rproflie and tests runed as expected

**Comment by radbasa** [comment]
@davidrsch This is fixed in `rhino@1.10.1` and `box.lsp@0.1.3`.

The change is made to the `.Rprofile.R` file, we wrap the `options()` inside a check if `box.lsp` is installed:

```R
if (nzchar(system.file(package = "box.lsp"))) {
  options(
    languageserver.parser_hooks = list(
      "box::use" = box.lsp::box_use_parser
    )
  )
}
```

Because the CI environment does not have a need for languageserver functions, `box.lsp` is not loaded in the CI environment.

**Comment by davidrsch** [comment]
Hello I tested and it worked ok. Not related to this issue, are there any plans to make `box.lsp` work in Positron IDE
